CONTRIBUTING.md

```diff
@@ -275,6 +275,12 @@ By giving feedback on this work (and related supporting work), you can help us m
 Finally, if nothing brings you more satisfaction than seeing every last issue labeled and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
 As discussed in [*How we're organized*](#how-were-organized), this role only requires good faith and a basic understanding of our development process.
 
+### How to adopt pull requests
+
+When you want to contribute pull request with the label *S-Adopt-Me*, to preserve the credit of the original authors, you must reference the PR hash in the description.
```

Comment:
Preferably adopters should build off of the existing commits of the original author. This should mention pulling down the commits of the original PR as a base and rebasing into `main` to preserve the commit history and credit of the original author.

Reply:
This is not always necessarily practical- the largest blocker to #5373 was a need for a rebase upon mainline, and squash-reduction to a single commit aided in that rebase significantly when creating #6853. If this is a preference, that's fine, but I'd avoid elevating it to a hard requirement.

Reply:
> This is not always necessarily practical- the largest blocker to #5373 was a need for a rebase upon mainline, and squash-reduction to a single commit aided in that rebase significantly when creating #6853. If this is a preference, that's fine, but I'd avoid elevating it to a hard requirement.

Hello @Dessix, did you comment in the correct PR? I didn't understand your comment...

Reply:
> Preferably adopters should build off of the existing commits of the original author.

@oCaioOliveira To rephrase my prior comment: "Reusing the original commits is sometimes hard".

Reply:
Yep, IMO this should be noted as a preference, not a requirement.